app/controllers/invitations_controller.rb

```diff
@@ -39,7 +39,8 @@ def create
       InvitationService.invite_to_group(recipient_emails: @invite_people_form.emails_to_invite,
                                         message: @invite_people_form.message_body,
                                         group: @invitable,
-                                        inviter: current_user)
+                                        inviter: current_user,
+                                        subject: params[:invite_people_form][:subject])
```

Comment:
i just tested it and the mail gem removes the special chars on its own, so i can't make header injection by sending a subject like this "a subject\r\nBcc: included@header.com", it turns out "a subject Bcc: included@header.com"